tests/test_render_outputs/test_merge_streams.xml

```diff
@@ -0,0 +1,23 @@
+<document source="merge_streams">
+    <CellNode cell_type="code" classes="cell">
+        <CellInputNode classes="cell_input">
+            <literal_block language="ipython3" linenos="False" xml:space="preserve">
+                import sys
+                print('stdout1', file=sys.stdout)
+                print('stdout2', file=sys.stdout)
+                print('stderr1', file=sys.stderr)
+                print('stderr2', file=sys.stderr)
+                print('stdout3', file=sys.stdout)
+                print('stderr3', file=sys.stderr)
+                1
+        <CellOutputNode classes="cell_output">
+            <literal_block classes="output stream" language="myst-ansi" linenos="False" xml:space="preserve">
```

Comment:
here they are all in single blocks

Reply:
ah - thanks @chrisjsewell that's helpful. So the order will always be the same (so long as the code-block contents doesn't change).

Reply:
yes the order of stdout / stderr, individually, are guaranteed, so they can be group merged. the order of stdout vs stderr is not guaranteed (they are independent streams), so stderr is placed always after stdout